docs/miner-guide.md

```
@@ -399,24 +377,17 @@ uv run kinitro miner commit \
 
 ### Commitment Format
 
-The commitment is stored on-chain as compact JSON to fit within chain limits:
+The commitment is stored on-chain in a compact format:
 
 **Plain commitment:**
-```json
-{"m":"your-username/kinitro-policy","r":"abc123def456...","d":"deployment-uuid"}
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)